calico_containers/calico_ctl/status.py

```diff
@@ -35,6 +36,13 @@ def status(arguments):
     this file's docstring with docopt
     :return: None
     """
+    try:
+        docker_client.version()
+    except ConnectionError:
```

Comment:
In this check you seem to be using "Is the Docker daemon running?" as a proxy for "Did the user start the node with `--runtime=none`?"  These aren't the same thing.  You could use `--runtime=none` even though Docker is running, for example. In that case, won't this function fail?